src/components/elements/ClickToShow.stories.tsx

```diff
@@ -10,3 +11,22 @@ export default {
 type Story = StoryObj<typeof ClickToShow>;
 
 export const Default: Story = { args: { children: 'Hidden Value' } };
+
+export const Clicked: Story = {
```

Comment:
When discussing this [ticket](https://github.com/open-path/hmis-accessibility/issues/55) we mentioned adding an a11y test in Storybook to ensure the correct behavior.

It looks like Storybook's actual [a11y testing](https://storybook.js.org/docs/writing-tests/accessibility-testing) is limited to Axe, i.e. we are limited to configuring which auto-checks from Axe get run. We can separately add interaction tests, however, like this one, but let's discuss the approach. Here I've done it by adding a separate story to check the behavior, but I remember at our design meeting about storybook structuring we wanted to err on the side of fewer stories.